test/inductor/test_torchinductor_dynamic_shapes.py

```diff
@@ -586,10 +585,10 @@ def fn(x):
         res0 = opt(x0)
         self.assertEqual(ref0, res0)
         # The second run triggers dynamic shapes.
-        x1 = torch.rand(8)
-        ref1 = fn(x1)
-        res1 = opt(x1)
-        self.assertEqual(ref1, res1)
+        # x1 = torch.rand(8)
+        # ref1 = fn(x1)
+        # res1 = opt(x1)
+        # self.assertEqual(ref1, res1)
```

Comment:
? can you add this back in ?

Reply:
ahh... It is mistakenly commented. uncommented.